doc-bridgepoint/notes/9564_feedback_on_formalized_relations/9564_feedback_on_formalized_relations.int.md

```diff
@@ -0,0 +1,149 @@
+---
+
+This work is licensed under the Creative Commons CC0 License
+
+---
+
+# Feedback for formalized associations  
+### xtUML Project Implementation Note
+
+### 1. Abstract
+
+This note describes the files changed for this work.
+
+### 2. Document References
+
+In this section, list all the documents that the reader may need to refer to.
+Give the full path to reference a file.  
+<a id="2.1"></a>2.1 [BridgePoint DEI #9564](https://support.onefact.net/issues/9564) Feedback on formalized relations  
+<a id="2.2"></a>2.2 [Design note](https://github.com/travislondon/bridgepoint/blob/9564_feedback_on_formalized_relations/doc-bridgepoint/notes/9564_feedback_on_formalized_relations/9564_feedback_on_formalized_relations.md) Project design note.  
+
+### 3. Background
+
+See [[2.2]](#2.2).  
+
+### 4. Requirements
+
+See [[2.2]](#2.2).  
+
+### 5. Work Required
+
+See [[2.2]](#2.2).  
+
+### 6. Implementation Comments
+
+None.  
+
+### 7. Unit Test
+
+See [[2.2]](#2.2).  
```

Comment:
Add a description of/pointer to the model created for this test.  Was a JUnit added?  If so, call it out.  Or, is it a manual test?